### PR TITLE
Fix plurality of records

### PR DESCRIPTION
**Pull request by sikai00**
Instead of '1 records' we should use '1 record'.

### CODE REVIEW

**Review by ThomasHoooo — DISMISSED** [review]
Do add in the abstracted method for UiPart as well

File: src/main/java/seedu/address/ui/ClientListPanel.java [hunk 1]
```diff
@@ -32,9 +32,20 @@ public ClientListPanel(ObservableList<Client> clientList) {
...
+        numClients.setText(numClientsString(clientList));
         clientList.addListener((ListChangeListener<? super Client>)
-                c -> numClients.setText(Integer.toString(clientList.size()) + " Records"));
+                c -> numClients.setText(numClientsString(clientList)));
+    }
+
+    /**
+     * Returns a string denoting the number of records for {@code Client} currently shown in the {@code clientList}.
+     */
+    private String numClientsString(ObservableList<Client> clientList) {
```
Comment (ThomasHoooo) [inline src/main/java/seedu/address/ui/ClientListPanel.java:43]: was thinking about making this a generic name, and then add into UiPart as an abstracted method also

Comment (sikai00) [reply src/main/java/seedu/address/ui/ClientListPanel.java:43]: hmm, this method does not belong to other Ui elements though

**Review by ThomasHoooo — APPROVED** [review]
LGTM

**Pull request merged**